src/MSBuild/Resources/Strings.resx

```diff
@@ -1037,7 +1037,7 @@
     <value>  -check
                      Enables BuildChecks during the build.
                      BuildCheck enables evaluating rules to ensure properties
-                     of the build. For more info see aka.ms/buildcheck
+                     of the build. For more info see https://aka.ms/buildcheck
```

Comment:
What does the team think about changing the wording a bit:

```diff
  -check
                     Run checks during build to catch common problems.
                     For more information, see: https://aka.ms/buildcheck
```

Or some other text that provides a sense of what BuildCheck is, without needing to follow a link.